application.js

```diff
@@ -0,0 +1,253 @@
+// 자주 사용되는 HTML 요소들을 저장해두기 위한 변수
+let newTodoInput = null
+let todoList = null
+let todoElements = null
+let filters = null
+let filterAll = null
+let filterActive = null
+let filterCompleted = null
+let filterContainer = null
+let selectedFilter = null
+// 자주 사용되는 객체, 상수값을 저장해두기 위한 변수
+let todoElementsNameArray = null
+const KEYWORD = "gTZ5JMw51a"
+
+window.onload = () => init()
+
+// 웹페이지 로드 시 실행되는 함수
+function init(){
+    // 초기화되지 않은 변수들을 HTML 요소로 초기화
+    newTodoInput = document.getElementById('new-todo-title')
+    todoList = document.getElementById('todo-list')
+    todoElements = todoList.children
+    filters = document.querySelector('ul.filters')
+    filterAll = filters.querySelector('li a.all')
+    filterActive = filters.querySelector('li a.active')
+    filterCompleted = filters.querySelector('li a.completed')
+    // 해당 변수들에 대한 적절한 이벤트 처리기 등록
+    newTodoInput.addEventListener('keyup', addNewTodo)
+    filterAll.addEventListener('click', filterAllViewChange)
+    filterActive.addEventListener('click', filterActiveViewChange)
+    filterCompleted.addEventListener('click', filterCompletedViewChange)
+    filterContainer = [filterAll, filterActive, filterCompleted]
+    selectedFilter = filterAll
+
+    // 저장된 할 일 항목이 있는지 확인, 있다면 불러오고 없다면 초기화.
+    todos = localStorage.getItem(KEYWORD)
+    if(todos == null){
+        todos = JSON.stringify([])
+        localStorage.setItem(KEYWORD, todos)
+    }
```

Comment:
```suggestion
    todos = JSON.parse(localStorage.getItem(KEYWORD)) ??  [];
```

널 병합 연산자를 사용하시면 이 부분을 더 깔끔하게 표현할 수 있을거 같습니다!

- [널 병합 연산자](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)

Reply:
널 병합 연산자란게 있는지 처음 알았네요. 상황에 따라 굉장히 유용하게 쓰일 것 같습니다.

review#9 커밋에서 반영하였습니다. 감사합니다.